### DISCUSSION TITLE
search option (l l -G) with result in context, instead of a list of commits is that contain such expression

### DISCUSSION THREAD

**Discussion by AtomicNess123** [magit/magit#4868] [Support] [answered]
I tried to find how to do this unsuccessfully, so it could be a good implementation:

When using the search option in MAGIT (l l -G) to search changes, the result is a list of commits that contain such expression. Is there a way to show the expression in context, as if it was a grep search? Otherwise, one must open each commit one by one.

**Comment by tarsius** [comment, marked as answer]
Use `- p` to enable the `--patch` argument.
